site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Kubernetes Maintenance"
+date: 2018-11-07T15:24:42+01:00
+weight: 10
+---
+
+This page helps developers access and perform any type of maintenance in the Production Mattermost Kubernetes Cluster, which is running on AWS using EKS.
+
+## Set up a local environment to access Kubernetes (K8s)
+
+1. Make sure you have `kubectl` version 1.10 or later installed. If not, follow [these instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+2. Use your OneLogin account to retrieve AWS Keys for the main Mattermost AWS account following [these instructions](../../onelogin-aws).
+
+Steps:
+
+- run `onelogin-aws-login` in the command line, it will ask your username, password and the two-factor auth.
+- select `arn:aws:iam::521493210219:role/OneLoginPowerUsers`
+- export the environment variable `AWS_PROFILE` which will be something like `521493210219/OneLoginPowerUsers/YOUR_EMAIL`
+
+3. Configure kubectl for Amazon EKS following [these instructions](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html).
```

Comment:
This isn't configuration it's about installing aws-iam-authenticator for Amazon EKS.

Reply:
updated